keyboards/tronguylabs/info.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
Still needs to be moved to `keyboards/tronguylabs/m122_3270/info.json`

Reply:
The readme, too?

Reply:
yeah, missed that one off somehow.